bedrock/mozorg/templates/mozorg/technology.html

```diff
@@ -35,6 +35,7 @@
 {% endblock %}
 
 {% block content %}
+{% include 'mozorg/developer/includes/sub-nav.html' %}
```

Comment:
And yeah, I know the content isn't really ideal for l10n. All the videos are English, images have text in them... ugh.